internal/resources/cluster/resource_cluster.go

```diff
@@ -263,7 +345,7 @@ func resourceClusterCreate(ctx context.Context, d *schema.ResourceData, m interf
 
 		log.Printf("[INFO] Applying %s cluster's deployment link manifest objects on to kubernetes cluster", constructFullname(d).ToString())
 
-		err = manifest.Create(k8sclient, deploymentManifests, true)
+		err = manifest.Create(*k8sclient, deploymentManifests, true)
```

Comment:
modify the manifest.Create function parameter to accept pointer k8sclient

Reply:
done